### PR TITLE
Setting up analytics (Piwik) #675

### PR DESCRIPTION
**Pull request by finnfiddle**
@iperdomo need to check Im doing things correctly on the backend for the env vars
@gabemart please check that you agree with how its been implemented
- [ ] **Update release notes if necessary**

### CODE REVIEW

File: backend/src/akvo/lumen/endpoint/env.clj [hunk 1]
```diff
@@ -8,6 +8,7 @@
       (cond-> {"keycloakClient" (:keycloak-public-client-id config)
                "keycloakURL" (:keycloak-url config)
                "flowApiUrl" (:flow-api-url config)
+               "piwikSiteId" (:piwik-site-id config)
```
Comment (dlebrero) [inline backend/src/akvo/lumen/endpoint/env.clj]: I dont see the value of piwik-site-id defined for production. Dont we want to do any tracking in production?

Comment (finnfiddle) [reply backend/src/akvo/lumen/endpoint/env.clj]: I think we need a different value for each tenant. how can this be done?

Comment (dlebrero) [reply backend/src/akvo/lumen/endpoint/env.clj]: If that is the case, the value should not come from the config but from the db.

Lets have a chat on Monday

**Comment by gabemart** [comment]
Frontend looks good to me 👍

**Pull request merged**